JPS_BASE_LIB/src/test/java/uk/ac/cam/cares/jps/base/config/test/KeyValueManagerTest.java

```diff
@@ -0,0 +1,61 @@
+package uk.ac.cam.cares.jps.base.config.test;
+
+import org.junit.Test;
+import uk.ac.cam.cares.jps.base.config.KeyValueManager;
+import static org.junit.Assert.*;
+
+import uk.ac.cam.cares.jps.base.config.KeyValueMap;
+import uk.ac.cam.cares.jps.base.exception.JPSRuntimeException;
+
+
+public class KeyValueManagerTest {
+
+    String testKey = "k1";
+    String testValue = "v1" ;
+
+    @Test
+    public void setTest(){
+
+        try{
+            KeyValueManager.set(testKey, testValue);
```

Comment:
This method attempts to send a HTTP request to a service which may or may not be running. Can this be avoided during testing by refactoring the code/using mocks?